src/views/catalog/wizard/tabs/yaml/components/YAMLEditor/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as React from 'react';
+
+export default React.lazy(() => import('./YAMLEditor'));
```

Comment:
why not under src/utils/components? this could be used by VM,VMI,Templates as well

Reply:
Yes absolutely right